tensorflow/compiler/tf2tensorrt/convert/convert_nodes_test.cc

```diff
@@ -2126,11 +2126,18 @@ TEST_F(OpConverterTest, ConvertActivation) {
         "Relu is only implemented for tensors, at my_act");
   }
 
+  const float ALPHA = 0.2f;
```

Comment:
nit: kAlpha to follow the naming in the style guide. We should even mark this as constexpr.
https://google.github.io/styleguide/cppguide.html#Constant_Names

Reply:
s/kALPHA/kAlpha? (Full words are never capitalized. Even abbreviations use Camel case with only the first letter capitalized.)
https://google.github.io/styleguide/cppguide.html#General_Naming_Rules

Reply:
Whoops, thanks for correcting!